Project.toml

```diff
@@ -7,6 +7,7 @@ version = "1.26.0"
 Calculus = "49dc2e85-a5d0-5ad3-a950-438e2897f1b9"
 Dierckx = "39dd38d3-220a-591b-8e3c-4c3a8c710a94"
 DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
+DiffEqFlux = "aae7a2af-3d4f-5e19-a356-7da93b79d9d0"
```

Comment:
I don't think we should take this on as a dependency...

Reply:
Definitely not taking that dependency here.